draft-ietf-quic-qpack.md

```diff
@@ -430,12 +431,9 @@ Dynamic table entries can have empty values.
 
 The size of the dynamic table is the sum of the size of its entries.
 
-The size of an entry is the sum of its name's length in bytes (as defined in
-{{string-literals}}), its value's length in bytes, and 32.
```

Comment:
The reference to {{string-literals}} here is misleading, because length in that section can mean length after Huffman encoding.